packages/esm-patient-forms-app/src/htmlformentry/html-form-entry-wrapper.scss

```diff
@@ -0,0 +1,8 @@
+.wrapper {
+  width: 95%;
+  border: none;
+  margin: 0;
+  padding: 0;
+  height: 3000px; // TODO: how to make this dynamic?
```

Comment:
My limited CSS skills were failing here... :) I basically want the iframe height to be equal to the height of the content it is displaying.

Reply:
I actually think this is quite hard! _Most_ DOM elements have some way of determining the height of their contents so you could just use `fit-content`, but iframes are special, because they can load cross-origin content. The best I can think of is adding to the `onLoad` function so it becomes something like:

```ts
const onLoad = useCallback(() => {
  const iframe = iframeRef.current;
  const dashboard = iframe.contentDocument;
  dashboard.querySelector('header')?.remove();
  dashboard.querySelector('.patient-header')?.remove();
  dashboard.querySelector('#breadcrumbs')?.remove();

  iframe.style.display = 'block';
  iframe.height = iframe.contentWindow.document.body.scrollHeight;
}, []);
```

Reply:
And this worked too, once I futzed around a bit and added a little padding... I will fiddle with this a little more....